x-pack/test/functional/apps/ml/data_visualizer/file_data_visualizer.ts

```diff
@@ -269,6 +269,9 @@ export default function ({ getService }: FtrProviderContext) {
 
           await ml.testExecution.logTestStep('imports the file');
           await ml.dataVisualizerFileBased.startImportAndWaitForProcessing();
+
+          await ml.testExecution.logTestStep('creates filebeat config');
+          await ml.dataVisualizerFileBased.selectCreateFileBeatConfig();
```

Comment:
Nit - `Filebeat` is one word, so this should be `selectCreateFilebeatConfig` I think.